GTG/gtk/application.py

```diff
@@ -335,8 +335,12 @@ def new_subtask(self, param, action):
 
     def add_parent(self, param, action):
         """Callback to add a parent to a task"""
-        if self.browser.have_same_parent():
-            self.browser.on_add_parent()
+        try:
+            if self.browser.have_same_parent():
+                self.browser.on_add_parent()
+
+        except IndexError:
```

Comment:
I think at least a comment on what this error causes would be nice, like `# When no task has been selected`.